### DISCUSSION TITLE
2024-06-12(수)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#136] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일

my profile 페이지 UI

### 해야할 일

인증 처리 API 연결 후 테스트
프로필 이미지 삽입 presigned URL 사용

### 어려운 일

인증

**Comment by devwqc** [comment]
### 완료한 일
- 모달 설계 작업

### 해야할 일
- 모달 레이아웃 작업
- 홈 페이지, 검색 페이지, 찜 페이지 작업

### 어려운 일
- 파이팅!!💪

**Comment by seolsis** [comment]
### 완료한 일
- 수정사항 반영
### 해야할 일
- toast, 하단 버튼 위치 조정
### 어려운 일

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
상품 상세 페이지

### 어려운 일
